core/task-impl.h

```diff
@@ -0,0 +1,13 @@
+#ifndef BESS_TASK_IMPL_H_
+#define BESS_TASK_IMPL_H_
+
+#include "task.h"
+
+#include "module.h"
+
+inline struct task_result Task::operator()(void)
```

Comment:
I cannot really include `module.h` in `task.h`, because `module.h` includes `packet.h`, which include `worker.h`.

This is the simplest way I found to break the loop.

Perhaps I can try to refactor other code to make this possible if you think it's better